Update readme for Kyma CLI (#362)
dgalbraith/chocolatey-packages: automatic/kyma-cli/README.md

> # [<img src="https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/icons/kyma-cli.png" width="48" height="48"/>Kyma CLI - Install, Manage and Test Kyma](<https://chocolatey.org/packages/kyma-cli>)

Reviewer: ```suggestion
# [<img src="https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/icons/kyma-cli.png" width="48" height="48"/>Kyma CLI - Install and Manage Kyma](<https://chocolatey.org/packages/kyma-cli>)

```

Author: should we also update it in our CLI docs? https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma.md

[![GitHub license](https://img.shields.io/github/license/kyma-project/kyma-cli)](https://github.com/kyma-project/cli/blob/master/LICENCE/)
[![Maintenance status](https://img.shields.io/badge/maintained%3F-yes-green.svg)](https://github.com/dgalbraith/chocolatey-packages/graphs/commit-activity)
[![AppVeyor build](https://img.shields.io/appveyor/ci/dgalbraith/chocolatey-packages)](https://ci.appveyor.com/project/dgalbraith/chocolatey-packages)
[![Software version](https://img.shields.io/badge/version-2.0.0-blue)](https://github.com/kyma-project/cli/releases/tags/2.0.0)
[![Chocolatey package version](https://img.shields.io/chocolatey/v/kyma-cli?label=Chocolatey)](https://chocolatey.org/packages/kyma-cli)

> Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install, manage, and test Kyma.

Reviewer: ```suggestion
Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install and manage Kyma.

```

> Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install, manage, and test Kyma.

Reviewer 2: ```suggestion
Kyma CLI is a command line tool that supports Kyma developers. It provides a set of commands you can use to install and manage Kyma.

```

Author: we are also using `which` instead of `that` in our CLI README https://github.com/kyma-project/cli/blob/main/README.md. I will update it there as well

## Features

* Generates and displays the bash or zsh completion script

> * Launches the Kyma dashboard in a browser window

Reviewer 2: ```suggestion
* Launches the Kyma Dashboard in a browser window

```

* Creates a system on the Kyma cluster with the specified name
* Deploys Kyma on a cluster based on the current or specified release

> * Imports certificates to local certificates storage or adds domains to the local host file

Reviewer 2: is that either certificates or domains, or can it do both at the same time? If both, then "and"

```suggestion
* Imports certificates to local certificates storage and adds domains to the local host file

```

Author: again this is how it is in our CLI docs: https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_import.md

Reviewer: That are two separate commands, so how to do that at the same time?

Author: We have these 2 features split into 2 different points now in the grouped features list

> * Provisions a new cluster on a platform of your choice. Currently, this command supports cluster provisioning on GCP, Azure, Gardener, and k3d

Reviewer 2: ```suggestion
* Provisions a new cluster on a platform of your choice (GCP, Azure, Gardener, and k3d)

```
If we have two full sentences, separated by a full stop, we need a full stop at the end as well. For consistency with the other lines, I'd rephrase it so we have a shorter phrase that can go without full stop.

* Shows the cluster version and the Kyma CLI version

> * Creates, applies and synchronizes local resources for you Function in the Kyma cluster

Reviewer 2: ```suggestion
* Creates, applies and synchronizes local resources for your Function in the Kyma cluster

```

> * Generates and displays the bash or zsh completion script
> * Launches the Kyma dashboard in a browser window
> * Creates a system on the Kyma cluster with the specified name
> * Deploys Kyma on a cluster based on the current or specified release
> * Imports certificates to local certificates storage or adds domains to the local host file
> * Provisions a new cluster on a platform of your choice. Currently, this command supports cluster provisioning on GCP, Azure, Gardener, and k3d
> * Shows the cluster version and the Kyma CLI version
> * Creates, applies and synchronizes local resources for you Function in the Kyma cluster
> * Runs a Function in Docker from local sources
> * [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests

Reviewer: ```suggestion
* Cluster Management
  * Installs and updates Kyma on a Kubernetes cluster supporting custom configuration
  * Shows the cluster version and the Kyma CLI version
* Local Environment
  * Provisions a local K3D cluster
  * Launches the Kyma dashboard in a browser window
  * Imports certificates to local certificates storage
  * Generates hosts file entries
* Serverless support
  * Creates, applies and synchronizes local resources to Functions in a Kyma cluster
  * Runs a Function in Docker from local sources
```

> * [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests

Reviewer 2: ```suggestion
* [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests.

```

![screenshot](https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/automatic/kyma-cli/screenshot.png)

## Prerequisites

> Kyma CLI requires the following software:

Reviewer 2: ```suggestion
To use Kyma CLI, you must install the following software:

```
